test/integration/topic-command-display.test.ts

```
@@ -0,0 +1,217 @@
+      .it('should display accounts commands correctly', (ctx) => {
+        expect(ctx.stdout).to.contain('Ably accounts management commands:');
+        expect(ctx.stdout).to.contain('ably accounts login');
+        expect(ctx.stdout).to.contain('ably accounts list');
+        expect(ctx.stdout).to.contain('ably accounts current');
+        expect(ctx.stdout).to.contain('ably accounts logout');
+        expect(ctx.stdout).to.contain('ably accounts switch');
+        expect(ctx.stdout).to.contain('ably accounts stats');
+        expect(ctx.stdout).to.contain('Run `ably accounts COMMAND --help`');
+        expect(ctx.stdout).not.to.contain('Example:'); // Examples only with --help
+      });
+  });
```

ESLint

[error] Do not pass arrow functions to describe()

(mocha/no-mocha-arrows)